app/views/notices/shop_employer_notices/0_welcome_notice_employer.html.erb

```diff
@@ -59,31 +59,13 @@
       <td>
         <ul style="list-style: circle solid;">
         <li class="notice-font">
-            <a href= "https://www.MAhealthconnector.org"><%= site_main_web_address %></a>
+            By calling <%=contact_center_phone_number%><br>
         </li>
         <li class="notice-font">
-             <a href= "#{site_main_web_address}"><%= site_main_web_address %></a>
+             TTY: <%=contact_center_tty_number%>
         </li>
         <li class="notice-font">
-            <a href= '#{site_main_web_address}'><%= site_main_web_address %></a>
-        </li>
-        <li class="notice-font">
-            <a href= "#{site_check}"><%= site_main_web_address %></a>
-        </li>
-        <li class="notice-font">
-            <%= link_to "#{site_main_web_address}", "#{site_check}" %>
-        </li>
-        <li class="notice-font">
-            <%= link_to "#{site_main_web_address}", "#{site_main_web_address}" %>
-        </li>
-        <li class="notice-font">
-            <%= link_to "#{site_main_web_address}", "#{site_check}", target: "_blank"%>
-        </li>
-        <li class="notice-font">
-            <%= mail_to "#{small_businesss_email}", "#{small_businesss_email}", target: "_blank"%>
-        </li>
-        <li class="notice-font">
-            <a mail_to= '#{small_businesss_email}'><%= small_businesss_email %></a>
+            Online at <%= link_to "#{site_main_web_address}", "#{site_main_web_address_url}" %>
```

Comment:
Yes the change is required as I had pushed some piece of code just to test which link works in Mozilla Firefox through QA. The changes need to be reverted.